rules/macos/process_creation/proc_creation_macos_csrutil_status.yml

```
@@ -0,0 +1,28 @@
+title: Configure System Integrity Protection (SIP) Enumeration
+id: 53821412-17b0-4147-ade0-14faae67d54b
+status: experimental
+description: Detects the use of csrutil to view the Configure System Integrity Protection (SIP) status. This technique is used in post-exploit scenarios. Also can be used to disable it.
+references:
+    - https://ss64.com/osx/csrutil.html
+    - https://www.virustotal.com/gui/file/05a2adb266ec6c0ba9ed176d87d8530e71e845348c13caf9f60049760c312cd3/behavior
+author: Joseliyo Sanchez, @Joseliyo_Jstnk
+date: 2023/12/26
+tags:
+    - attack.discovery
+    - attack.t1518.001
+logsource:
+    product: macos
+    category: process_creation
+detection:
+    # VT Query: behavior_processes:"csrutil status" p:5+ type:mac
+    selection_image:
+        Image|endswith:
+            - '/usr/bin/csrutil'
+            - '/csrutil'
```

```suggestion
        Image|endswith: '/csrutil'
```
`/csrutil` cover `/usr/bin/csrutil`